lib/graphql.ts

```diff
@@ -17,16 +17,21 @@ export const client = new ApolloClient({
   link: ApolloLink.from([httpLink as any as ApolloLink]),
 })
 
-// NOTE: https://thegraph.com/hosted-service/subgraph/balancer-labs/balancer
-export const POOL_TOKEN_BALANCES = gql`
-  {
-    pool(
-      id: "${BPT_POOL_ID}"
-    ) {
-      tokens {
-        symbol
-        balance
+export async function fetchPoolTokenBalances() {
```

Comment:
If we don't have any usage for Apollo elsewhere, you may want to consider `graphql-request` as a more lightweight solution.